docs/api/model.rst

```diff
@@ -20,6 +20,7 @@ Language Modeling
     standard_lstm_lm_1500
     StandardRNN
     get_model
+    BigRNN
```

Comment:
sparse?

Reply:
it's special in that it uses sparse blocks in the output. the embedding size/hidden size etc won't necessarily be large.